src/dk.nita.saml20/dk.nita.saml20/Config/AppSwitchReturnURL.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Xml.Serialization;
+
+namespace dk.nita.saml20.config
+{
+    /// <summary>
+    /// Configuration element for app switch return url.
+    /// </summary>
+    public class AppSwitchReturnURL 
+    {
+        /// <summary>
+        /// AppSwitch platform. Can be either Android or iOS.
+        /// </summary>
+        [XmlAttribute(AttributeName = "platform")]
+        public string Platform;
```

Comment:
Why isn't this an enum value? Would make logic in the FindAppSwitchReturnUrlForPlatform-method simpler and the value is already suppose to be a known set of values.

I would discourage the use of string comparison in this situation.